meetings/2024-06-06/minutes.md

```diff
@@ -0,0 +1,154 @@
+* Updates from related communities:
+  * [PQCA](https://github.com/PQCA)
+  * [Open Quantum Safe](https://github.com/open-quantum-safe)
+* Discussion - [What does 'assured' mean?](https://github.com/pq-code-package/tsc/issues/3)
+* Discussion - [Common API](https://github.com/pq-code-package/tsc/issues/4)
+* Review status of sub projects:
+  * [mlkem-c-generic](https://github.com/pq-code-package/mlkem-c-generic)
+  * [mlkem-c-embedded](https://github.com/pq-code-package/mlkem-c-embedded)
+  * [mlkem-c-aarch64](https://github.com/pq-code-package/mlkem-c-aarch64)
+  * [mkkem-libjade](https://github.com/pq-code-package/mlkem-libjade)
+  * [mlkem-rust-libcrux](https://github.com/pq-code-package/mlkem-rust-libcrux)
+  * [documentation](https://github.com/pq-code-package/documentation)
+* Review  [open TSC issues](https://github.com/pq-code-package/tsc/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc):
+
+  * any noteable progress
+  * topics for next meeting
+* Any other business
+
+## Introductions / Welcome
+
+Apologies noted from Douglas, Matthias and Franziskus.
+
+JP Lomas introduced himself as a developer with QRL Foundation.
+Jonas Schneider-Bensch introduced himself as a developer from Crispin, and providing cover for Franziskus who can't make it today.
+
+
+## Decisions
+
+no formal votes
+
+## Discussion
+
+### Minutes/actions from previous meeting
+
+Outstanding issues: Will close API discussion action as we started that discussion.
+Minutes were approved via github.
+
+### Agree ongoing meeting schedule
+
+We agreed not enough of the voting members were present, so we will continue on the existing schedule for now. (every two weeks, same time)
+
+### Updates from related communities
+
+Ongoing discussions at PQCA
+
+* Security workgroup
+* CBOM workgroup
+
+ OQS
+
+* new release coming out
+* some vulnarabilities reported ie compiler optimization bug. Yarkin mentioned there was also an outstanding discussion in oqs about row-hammer attach on ML-KEM, and whether such attached featured in the threat model. Ry mentioned that if anyone wanted to see the draft advisories, they should contact him for access.
+
+### What does assured mean
+
+* See [pq-code-package/documentation#8](https://github.com/pq-code-package/documentation/pull/8) for initial draft of docs
+* Hanno: need more refinement about formal proof. Add more links ie to aarch64 repo. Long term strive for functional correctness of the assembly. For example assembley level guarantees, constant time, trusted computing base assumptions.
+* Tiego - will look into discussion and see how he can contribute from the jasmine side, along with Manuel.
+
+### Common API
+
+* Tiago/ Hanno- general agreement with proposal. Funcion names should be namespaced (same signature) as multiple parts of code package could be used together
+* Should _derand functions be exposed to user? Diedre said that NIST explicitly stated redrand should not be part of full public api
```

Comment:
```suggestion
* Should _derand functions be exposed to user? Deirdre said that NIST explicitly stated redrand should not be part of full public api
```

Reply:
Thanks for that correction. I thought I'd already spotted/corrected but it slipped through